pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -83,6 +85,10 @@ public static String constructPropertyStorePathForKafkaPartitions(String realtim
     return StringUtil.join("/", PROPERTYSTORE_KAFKA_PARTITIONS_PREFIX, realtimeTableName);
   }
 
+  public static String constructPropertyStorePathForInstancePartitions(String offlineTableName) {
```

Comment:
This mapping table is used for offline segments only. In terms of routing, the broker will look at `KAFKA_PARTITIONS` and pick one server for a partition for LLC segments (in my understanding).